scripts/common/common.sh

```diff
@@ -1036,3 +1036,52 @@ function canonical_image_name() {
     fi
     echo "$image"
 }
+
+# wait_for_running_pods waits for pod(s) in a given namspace to transition to Running status
+function wait_for_running_pods() {
```

Comment:
does this function really _wait_ for anything?
Or is it intended to be used with a spinner?

If it's intended to be used with a spinner, it shouldn't be logging things IIRC - that messes up the spinner

Reply:
It's meant to be used as an arg to `spinner_until()`. I'll change the name and remove the logs.